app/views/publish/courses/_description_status_panel.html.erb

```diff
@@ -0,0 +1,40 @@
+<div class="app-status-box">
+  <h3 class="govuk-heading-m">Status</h3>
+  <p class="govuk-body" data-qa="course__content-status">
+    <%= course.status_tag %>
+  </p>
+  <h3 class="govuk-heading-s govuk-!-margin-bottom-0">
+    <% if @recruitment_cycle.current_and_open? %>Is it<% else %>Will it be<% end %> on <abbr class="app-!-text-decoration-underline-dotted" title="Find postgraduate teacher training">Find</abbr>?
+  </h3>
+  <p class="govuk-body" data-qa="course__is_findable">
+    <%= course.on_find %>
+  </p>
+<!-- Do we need a default here? -->
```

Comment:
I think the default was the `unpublished` beforehand


https://github.com/DFE-Digital/publish-teacher-training/blob/master/app/views/courses/_status_panel.html.erb#L13-L19

Reply:
Yes, but the current change is due to the logic chain in `is_published?`. We have to check `unpublished` first in the `if` block, just wondering if I am allowing a null condition to creep in? Or do we catch all possible conditions as is?

Reply:
Have set the default, looks a bit quirky but there you go...